### PR TITLE
Translation cleanups (in principle)

### PR DESCRIPTION
**Pull request by UncombedCoconut**
This actually drops a working translation in order to use ngettext(), but should otherwise be a win. Assumption: it's worth the work and the translations need maintenance anyway.

### CODE REVIEW

File: dat/scripts/numstring.lua [hunk 1]
```diff
@@ -45,6 +45,18 @@ function tonnestring( tonnes )
...
 
+--[[
+-- @brief Like tonnestring, but for abbreviations.
+--
+--    @param tonnes Number of tonnes.
+--    @return A short string like "22t" describing the given mass.
+--]]
+function tonnestring_short( tonnes )
+   -- Translator note: this form represents an abbreviation of "_ tonnes".
+   return gettext.ngettext( "%dt", "%dt", tonnes ):format( tonnes )
```
Comment (bobbens) [inline dat/scripts/numstring.lua:56]: Shouldn't it be `"%d t"`?

**Review by bobbens — COMMENTED** [review]
Other than the small space it looks good.

**Comment by UncombedCoconut** [comment]
The only reason it isn't, is that it previously wasn't. Let's do it!

**Comment by bobbens** [comment]
Thanks!

**Pull request merged**